File: app/src/main/java/com/blackmirror/dongda/ui/activity/MyBrandActivity.java
package com.blackmirror.dongda.ui.activity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.blackmirror.dongda.R;
import com.blackmirror.dongda.ui.base.BaseActivity;
import com.facebook.drawee.view.SimpleDraweeView;

public class MyBrandActivity extends BaseActivity {

    private ImageView iv_back;
    private SimpleDraweeView sv_brand_photo;
    private TextView tv_brand_name;
    private TextView tv_about_brand;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected int getLayoutResId() {
        return R.layout.activity_my_brand;
    }

    @Override
    protected void initInject() {

    }

    @Override
    protected void initView() {
        iv_back = findViewById(R.id.iv_back);
        sv_brand_photo = findViewById(R.id.sv_brand_photo);
        tv_brand_name = findViewById(R.id.tv_brand_name);
        tv_about_brand = findViewById(R.id.tv_about_brand);
    }

    @Override
    protected void initData() {

    }

    @Override
    protected void initListener() {
        iv_back.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: app/src/main/java/com/blackmirror/dongda/ui/activity/MyBrandActivity.kt
package com.blackmirror.dongda.ui.activity

import android.widget.ImageView
import android.widget.TextView
import com.blackmirror.dongda.R
import com.blackmirror.dongda.ui.base.BaseActivity
import com.facebook.drawee.view.SimpleDraweeView

class MyBrandActivity : BaseActivity() {

    private lateinit var iv_back: ImageView
    private lateinit var sv_brand_photo: SimpleDraweeView
    private lateinit var tv_brand_name: TextView
    private lateinit var tv_about_brand: TextView

    override val layoutResId: Int
        get() = R.layout.activity_my_brand

    override fun initInject() {

    }

    override fun initView() {
        iv_back = findViewById(R.id.iv_back)
        sv_brand_photo = findViewById(R.id.sv_brand_photo)
        tv_brand_name = findViewById(R.id.tv_brand_name)
        tv_about_brand = findViewById(R.id.tv_about_brand)
    }

    override fun initData() {

    }

    override fun initListener() {
        iv_back.setOnClickListener { finish() }
    }
}
